components/calendar/Cargo.toml

```diff
@@ -28,13 +28,13 @@ all-features = true
 
 [dependencies]
 displaydoc = { version = "0.2.3", default-features = false }
-icu_provider = { version = "1.0.0", path = "../../provider/core", features = ["macros"] }
-icu_locid = { version = "1.0.0", path = "../../components/locid" }
-tinystr = { version = "0.7", path = "../../utils/tinystr", features = ["alloc", "zerovec"], default-features = false }
-zerovec = { version = "0.9", path = "../../utils/zerovec", default-features = false, features = ["derive"] }
-writeable = { version = "0.5", path = "../../utils/writeable" }
+icu_provider = { path = "../../provider/core", features = ["macros"] }
```

Comment:
Question: How are you able to remove these versions? I think `cargo publish` doesn't let you publish unless all of your `dependencies` have a version. Is this `cargo workspaces publish` magic?

Reply:
I *think* `cargo workspaces` allows this. I'll know when I try.